creator-node/src/blacklistManager.js

```diff
@@ -8,103 +7,201 @@ const REDIS_SET_BLACKLIST_USERID_KEY = 'SET.BLACKLIST.USERID'
 const REDIS_SET_BLACKLIST_SEGMENTCID_KEY = 'SET.BLACKLIST.SEGMENTCID'
 
 class BlacklistManager {
-  static async blacklist (ipfs) {
+  static async init (ipfs) {
```

Comment:
tbh we dont need `ipfs` anymore since we only used it previously to unpin content. but we dont pin content anymore